e2e.config.cjs

```diff
@@ -0,0 +1,10 @@
+module.exports = {
```

Comment:
This configuration needs to be on the same level as the package.json

I decided to use a different format for e2e tests (`e2e.js`) to not be picked up by normal ava that can be run by `npx ava` as these tests require the npm registry that is only spawned by running `npm run e2e`